lib/models/game.dart

```diff
@@ -13,6 +13,8 @@ class Game {
   final int boundarySize;
   final GamePhase phase;
 
+  double distanceFromUser;
```

Comment:
this is not final because when we create the game object (after getting data from firestore) we dont know the distance until we get the users location and calculate distanceFromUser on the client

Reply:
if thats a bit hard to understand (because of my poor articulation) happy to chat it through :)